plugin/diagnostics.py

```diff
@@ -252,7 +252,7 @@ def end_file(self, file_name: str) -> None:
         self._relevant_file = False
 
     def end(self) -> None:
-        for severity in range(DiagnosticSeverity.Error, DiagnosticSeverity.Hint + 1):
+        for severity in range(settings.show_diagnostics_severity_level + 1):
```

Comment:
This is `show_diagnostics_severity_level`, but in LSP.sublime-settings you changed `auto_show_diagnostics_panel_level`?

The `show_diagnostics_severity_level` is still at `2`. I think you meant to change that?

Reply:
I made a mistake, I didn't intend to change the level for the diagnostic panel. I will update that. 
Thanks :)